acn.py

```diff
@@ -0,0 +1,66 @@
+import bpy
+from dmx.sacn import sACNreceiver
+from dmx.data import DMX_Data
+from dmx.logging import DMX_LOG
+
+
+class DMX_ACN:
```

Comment:
Is there a reason this is called `ASN` instead of `sACN`? Might be a little confusing.

Reply:
Good point. I keep the acn.py as is to prevent namespace collision, as the module is sacn already. I did not like pyacn.py or pythonacn.py... but if you have suggestion, i don't mind renaming the file.

Reply:
Nothing comes to mind right now.. if we stumble upon it on the future we can change it.